src/main/java/me/leoko/advancedban/Universal.java

```diff
@@ -57,16 +57,15 @@ public void setup(MethodInterface mi) {
 
         mi.setCommandExecutor("advancedban");
         mi.setCommandExecutor("change-reason");
-        mi.setCommandExecutor("ban");
-        mi.setCommandExecutor("tempban");
-        mi.setCommandExecutor("ipban");
-        mi.setCommandExecutor("ban-ip");
-        mi.setCommandExecutor("banip");
-        mi.setCommandExecutor("tempipban");
-        mi.setCommandExecutor("tipban");
-        mi.setCommandExecutor("banip");
-        mi.setCommandExecutor("kick");
-        mi.setCommandExecutor("warn");
+        if(!config.getstring(Commands.ban).equals""){mi.setCommandExecutor(config.getstring(commands.ban));}
```

Comment:
That's not even valid Java syntax. You've created the modifications in GitHub's web UI. Have you tested them?